### ISSUE TITLE
Getting a strange dependency error

### ISSUE THREAD

**Issue by homeslicesolutions** [privateOmega/html-to-docx#20]
```
Failed to compile.

/Users/jvu/dev/athena/node_modules/libtidy-updated/node_modules/node-pre-gyp/lib/info.js
Module not found: Can't resolve 'aws-sdk' in '/Users/jvu/dev/athena/node_modules/libtidy-updated/node_modules/node-pre-gyp/lib'
```

I have 2 issues with this library in terms of dependencies.  First of, working locally works just fine and works perfectly, but when injected in our build process it runs to several problems.  We have a NODE application that I have this installed in which causes some conflict in Node version due to a deep dependency of `node-pre-gyp` which basically manages node of that particular machine.  That deep dependency is derived from `libtidy-updated` which it seems like you use to validate and clean up HTML string.  I had to fork this repo and remove the filter and it worked perfectly.  So I'm unsure how you want to address this, maybe have HTML cleanup a responsibility for the user OR have it be optional.  Secondly, because of these dependencies, this code DOES NOT work on browser since it's requiring node and npm.  Maybe there's a different Tidy/Minifier filter you can find or maybe include detailed instructions on how to manually tidy up HTML for the user and have it work that way.

**Comment by privateOmega** [comment]
@homeslicesolutions Yup these all arose from libtidy-updated and that is the menace, I guess best is to remove that altogether and then make this library compatible with browser eventually. Feel free to raise a PR to develop branch.
@amrita-syn Thoughts?

**Comment by amrita-syn** [comment]
Makes sense. Anyway this repo should expect 'clean' html and user can ensure cleanliness through a method of their choice. We should specify the HTML standard that is required for correct file generation and leave it at that. Examples with and without libtidy can be shown for those who would want to use it. 

@privateOmega We will have to modify the way we are using the package to accommodate these changes.

**Comment by privateOmega** [comment]
@amrita-syn Exactly my thought too. :+1:

**Comment by homeslicesolutions** [comment]
@privateOmega , I also noticed that you use `xmlbuilder2` which is also a node.js library.  When tried to use it in the browser in my React application, I get the following error 
```
Uncaught (in promise) TypeError: Cannot call a class as a function
    at e.exports (classCallCheck.js:3)
    at new r (NodeImpl.ts:93)
    at r.<anonymous> (createSuper.js:14)
    at new r (DocumentImpl.ts:69)
    at Object.t.create_document (CreateAlgorithm.ts:61)
    at new r (WindowImpl.ts:28)
    at Function.value (WindowImpl.ts:41)
    at Object.t.create_window (CreateAlgorithm.ts:47)
    at e.get (DOMImpl.ts:68)
    at new e (DOMImplementationImpl.ts:27)
```

**Comment by privateOmega** [comment]
@homeslicesolutions @amrita-syn I have created a PR which removes libtidy altogether from html-to-docx side, so I guess people who would want to tidy HTML input before it comes into html-to-docx can use their own preferred html cleaner package.

**Comment by privateOmega** [comment]
@homeslicesolutions Published new version v1.1.31, which essentially removes libtidy from it, hence wouldn't cause anymore of the weird dependency issues.
Also I had created an issue on XMLBuilder2 repo, but not sure how I would create a test case for it.
Closing the current issue since the issue in scope is resolved.
